test/watcher_test.js

```diff
@@ -29,11 +29,9 @@ describe('Watcher', function() {
     watched: true,
   };
 
-  const buildResults = {};
-
   const builder = {
-    async build() {
-      return buildResults;
+    async build(_, buildAnnotation) {
```

Comment:
We should echo the buildAnnotation we where provided, that ensures this test suite can be useful